### PR TITLE
feat: add admin endpoints

### PR DESCRIPTION
**Pull request by kneal**
This PR adds all the [admin](https://github.com/go-vela/server/tree/master/api/admin) endpoints to the SDK to make them available to admins who need to script against the endpoints.

The following is example syntax to access the admin endpoints on a client:

```go
client.Admin.Build.GetAll(nil)
```

_Note: GoDoc examples was not added at this time but can be if we think it's valuable._

Dependent PRs:
https://github.com/go-vela/mock/pull/75

fixes: go-vela/community#100

### CODE REVIEW

**Review by wass3r — COMMENTED** [review]
conflicts need to be resolved - not sure if you noticed.

File: vela/admin.go [hunk 1]
```diff
@@ -0,0 +1,346 @@
...
+
+import (
+	"fmt"
+
+	"github.com/go-vela/types/library"
+)
+
+type (
+	// AdminService handles retrieving builds from
+	// the server methods of the Vela API.
```
Comment (jbrockopp) [inline vela/admin.go:15]:
```suggestion
	// AdminService handles retrieving resources from
	// the server methods of the Vela API.
```

**Review by jbrockopp — CHANGES_REQUESTED** [review]
One small typo 👍

**Review by jbrockopp — APPROVED** [review]
LGTM

**Review by meinenec — APPROVED** [review]

**Pull request merged**